stdlib/public/Concurrency/Actor.cpp

```diff
@@ -1752,10 +1752,20 @@ void swift::swift_defaultActor_deallocateResilient(HeapObject *actor) {
                       metadata->getInstanceAlignMask());
 }
 
-// TODO: most likely where we'd need to create the "proxy instance" instead? (most likely remove this and use swift_distributedActor_remote_create instead)
-void swift::swift_distributedActor_remote_initialize(DefaultActor *_actor) { // FIXME: remove distributed C++ impl not needed?
-  auto actor = asImpl(_actor);
-  actor->initialize(/*remote=*/true);
+// TODO: rename initialize to allocate throughout htis built-in. Delete `swift_distributedActor_remote_create` throughout.
+OpaqueValue* swift::swift_distributedActor_remote_initialize(Metadata *actorType) {
```

Comment:
>  error: out-of-line definition of 'swift_distributedActor_remote_initialize' does not match any declaration in namespace 'swift'

Didn't have time to dig into why.

Reply:
It has a definition in  a header file as well, there it was still DefaultActor, easily fixed :)

Reply:
Okey so this is the "create", if we are to follow concurrency lib convetions, where "initialize" used to be "there is some memory, initialize _into it_", like the actor does. Impl looks nice though - very excited if this is enough to make it work ❤️